angular/src/directives/control-value-accessors/value-accessor.ts

```diff
@@ -12,6 +12,13 @@ export class ValueAccessor implements ControlValueAccessor {
   constructor(protected el: ElementRef) {}
 
   writeValue(value: any) {
+    /**
+     * TODO for Ionic 6:
+     * Change `value == null ? '' : value;`
+     * to `value`. This was a fix for IE9, but IE9
+     * is no longer supported; however, this change
+     * is potentially a breaking change
+     */
```

Comment:
It is possible to trigger a deprecation warning if value == null? (console.warn)

Reply:
There isn't anything to deprecate here as we are just updating the default value for `null`. In Ionic 6 the `null` default would just be changing from `''` to `null`. We will make note of it in the breaking changes document.